src/ucp/core/ucp_mm.c

```diff
@@ -178,20 +183,107 @@ static ucs_status_t ucp_mem_alloc(ucp_context_h context, size_t length,
 }
 
 
+static inline unsigned uct_flags(ucp_mem_map_params_t *params)
+{
+    unsigned flags = 0;
+
+    if (params && (params->field_mask & UCP_MEM_MAP_PARAM_FIELD_FLAGS)) {
+        if (params->flags & UCP_MEM_MAP_NONBLOCK) {
+            flags |= UCT_MD_MEM_FLAG_NONBLOCK;
+        }
+
+        if (params->flags & UCP_MEM_MAP_FIXED) {
+            flags |= UCT_MD_MEM_FLAG_FIXED;
+        }
+    }
+
+    return flags;
+}
+
+/* Matrix of behavior
```

Comment:
As I understand `verbatim` does not support hyperlinks, doesn't it? The resulting table includes links to all actual fields and flags.